spec/outputs/dynatrace_spec.rb

```diff
@@ -201,17 +201,6 @@
       end
     end
 
-    context 'with more than 4.5MB of events' do
```

Comment:
Tests removed because they duplicate other tests which use a smaller default. We're not testing the default value but I don't think there's a real need to. We test the `max_payload_size` behavior.

Reply:
I'd be fine with keeping the tests based on the default, and just updating them once now based on the new value. Either is fine though.